torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -1158,13 +1159,16 @@ def _center_crop_compute_crop_anchor(
 
 def center_crop_image_tensor(image: torch.Tensor, output_size: List[int]) -> torch.Tensor:
     crop_height, crop_width = _center_crop_parse_output_size(output_size)
-    image_height, image_width = get_spatial_size_image_tensor(image)
+    shape = image.shape
+    if image.numel() == 0:
+        return image.reshape(shape[:-2] + (crop_height, crop_width))
```

Comment:
This is needed because the original `pad_image_tensor` method below had a mitigation for zero batch images. So I thought to hit that point early.